apps/shipments/iot_client.py

```diff
@@ -36,3 +42,93 @@ def update_shadow(self, device_id, shadow):
             raise AWSIoTError("Error in response from AWS IoT")
 
         return iot_shadow['data']
+
+    def get_list_owner_devices(self, owner_id, next_token=None, results=None, **kwargs):
+        LOG.debug(f'Getting devices for: {owner_id} from AWS IoT')
+        log_metric('transmission.info', tags={'method': 'DeviceAWSIoTClient.get_list_owner_devices'})
+
+        active = kwargs.get('active', None)
+        in_bbox = kwargs.get('in_bbox', None)
+
+        if in_bbox:
+            in_bbox = ','.join([c.strip() for c in in_bbox.split(',')])
+            in_bbox_is_valid, message = self.validate_in_bbox(in_bbox)
+            if not in_bbox_is_valid:
+                raise ParseError(message)
+
+        if not next_token:
+            results = []
+
+        params_dict = OrderedDict(ownerId=owner_id,
+                                  maxResults=settings.IOT_DEVICES_MAX_RESULTS,
+                                  in_bbox=in_bbox if in_bbox else '',
+                                  nextToken=next_token if next_token else '',)
+
+        list_devices = self._get(f'devices?{urlencode(params_dict)}')
+
+        if 'error' in list_devices:
+            raise AWSIoTError("Error in response from AWS IoT")
+
+        new_devices = list_devices['data'].get('devices', None)
+        if new_devices:
+            results.extend(new_devices)
+
+        next_token = list_devices['data'].get('nextToken', None)
+        if next_token:
+            return self.get_list_owner_devices(owner_id, next_token=next_token, results=results, active=active,
+                                               in_box=in_bbox)
```

Comment:
this is posting in_box instead of in_bbox

Reply:
Good catch this should be `in_bbox`

Reply:
:+1: